app/kumactl/cmd/install/install_logging.go

```diff
@@ -1,18 +1,57 @@
+// Deprecated: `kumactl install logging` is deprecated, use `kumactl install observability` instead
+
 package install
 
 import (
 	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
+
+	kumactl_data "github.com/kumahq/kuma/app/kumactl/data"
+	kumactl_cmd "github.com/kumahq/kuma/app/kumactl/pkg/cmd"
+	"github.com/kumahq/kuma/app/kumactl/pkg/install/data"
+	"github.com/kumahq/kuma/app/kumactl/pkg/install/k8s"
 )
 
-func newInstallLogging() *cobra.Command {
+type loggingTemplateArgs struct {
+	Namespace string
+}
+
+func newInstallLogging(pctx *kumactl_cmd.RootContext) *cobra.Command {
+	args := pctx.InstallLoggingContext.TemplateArgs
 	cmd := &cobra.Command{
 		Use:   "logging",
 		Short: "Install Logging backend in Kubernetes cluster (Loki)",
 		Long:  `Install Logging backend in Kubernetes cluster (Loki) in its own namespace.`,
 		RunE: func(cmd *cobra.Command, _ []string) error {
-			return errors.New("we're migrating to `observability`, please use `install observability`")
+			_, _ = cmd.ErrOrStderr().Write([]byte("we're migrating to `observability`, please use `install observability`"))
```

Comment:
There's `cobra.Command.Deprecated` maybe that would be a good way to do this and it might also add info to the docs we generate which would be cool

Reply:
done